tasks/manage.yml

```diff
@@ -0,0 +1,32 @@
+- block:
+  - name: Export Django project's required environment variables
+    shell: "export {{ item.name }}={{ item.value }}"
```

Comment:
The goal of this line is to set environment variable for migrate and collectstatic actions below? I'm surprised that it works at all and I'm not sure that ansible guarantees that all tasks from a given role are executed in the same shell session. Shouldn't we use something like http://docs.ansible.com/ansible/playbooks_environment.html instead?